AppSyncRealTimeClient/Support/SubscriptionConstants.swift

```diff
@@ -13,6 +13,8 @@ public struct SubscriptionConstants {
     public static let appsyncRealtimeHostPart = "appsync-realtime-api"
 
     public static let realtimeWebsocketScheme = "wss"
+    
+    public static let appsyncCustomDomainWebSocketPathAppend = "realtime"
```

Comment:
so this is only added to the end of the URL for the custom domains?

Reply:
yes, this is added only subscriptions